### DISCUSSION TITLE
Is it possible to use Timber with custom tables ?

### DISCUSSION THREAD

**Discussion by benjaminviaud** [timber/timber#2927] [Q&A]
Hi everyone,

Is it possible to use Timber with custom tables ?

For example:
- if we have "products" tables only;
- and also if we have "products" and "productmeta". _For many reasons we don't want to use posts and postmeta tables._

Thanks for your reply.

Have nice day.

**Comment by nlemoine** [comment]
Please do not bypass the issue template recommandations. Use issue for their primary purposes (actual issues). This helps us a lot in our triage/bug resolution work. 

Thanks you for understanding 🙂
